README.md

```diff
@@ -9,6 +9,8 @@ This role is needed to get access for Extra Packages for Enterprise Linux (EPEL)
 
 ## Installation
 
+**NOTE**: The armhfp repository get automatically rebuilded without any QA and tests. If you want to use it set `wtd_repo_epel_armhfp_confirm` to `true`.
```

Comment:
I would change it to something like: 

The armhfp repository is automatically rebuilt without any QA or tests.
Please set `wtd_repo_epel_armhfp_confirm` to `true`